src/components/ButtonPanel.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Button from './Button';
+
+const ButtonPanel = props => ( // eslint-disable-line
+  <div>
+    <div>
+      <Button buttonName="AC" />
+      <Button buttonName="+/-" />
+      <Button buttonName="%" />
+      <Button buttonName="÷" />
+    </div>
+    <div>
+      <Button buttonName="7" />
+      <Button buttonName="8" />
+      <Button buttonName="9" />
+      <Button buttonName="X" />
+    </div>
+    <div>
+      <Button buttonName="4" />
+      <Button buttonName="5" />
+      <Button buttonName="6" />
+      <Button buttonName="-" />
+    </div>
+    <div>
+      <Button buttonName="1" />
+      <Button buttonName="2" />
+      <Button buttonName="3" />
+      <Button buttonName="+" />
+    </div>
+    <div>
+      <Button buttonName="0" />
+      <Button buttonName="." />
+      <Button buttonName="=" />
+    </div>
+  </div>
+);
```

Comment:
**[ OPTIONAL ]** We could implement a dynamic way of rendering this particular component instead of hardcoding the buttonName prop and the div structure of it. Think of creating an array with the values of the buttonName prop and rendering it using a map loop, that way we can reduce the number of lines in the code and it will look better.